include/umf/memory_pool_ops.h

```diff
@@ -185,7 +188,24 @@ typedef struct umf_memory_pool_ops_t {
     /// @return UMF_RESULT_SUCCESS on success or appropriate error code on
     ///         failure.
     ///
+
     umf_result_t (*ext_trim_memory)(void *pool, size_t minBytesToKeep);
+    ///
+    /// @brief Post-initializes and set up memory pool.
+    ///
+    /// \details
+    /// * This function *must* be implemented if the pool/provider supports CTL that overrides defaults.
+    /// * This function *must* free any resources allocated in the function.
+    /// * This function *must* be called after the memory pool has been allocated in initialize function
```

Comment:
`This function *must* be called after the memory pool has been allocated in initialize function`

Must be called by whom? If UMF calls it, then the comment is missleading.

Reply:
It's about the function order. Generally, this function is the same case as `initialize()`, it's also invoked in the same way - internally.